.changeset/soft-buses-fold.md

```diff
@@ -0,0 +1,125 @@
+---
+'@spectrum-web-components/infield-button': patch
+'@spectrum-web-components/number-field': patch
+'@spectrum-web-components/textfield': patch
+'@spectrum-web-components/search': patch
+'@spectrum-web-components/styles': patch
+---
+
+# Release Note
+
```

Comment:
These get added to the Changelogs which already have an h1 heading so I'd suggest leaving this off and starting headings at h3 (the tags are listed as h2s)